modules/dhp_connected_devices/spec/lib/fitbit_client_spec.rb

```diff
@@ -62,4 +81,20 @@
       expect(subject).to be_an_instance_of(described_class)
     end
   end
+
+  describe 'get_auth_code' do
+    it 'returns code as a string when auth code given' do
+      expect(subject.get_auth_code({ code: 'code' })).to eq('code')
```

Comment:
NIT: can we give a different value? e.g. 'code_value' or '12345'. Might make this test a bit clearer

Reply:
Definitely, updating